lib/drivers/carelinkDriver.js

```diff
@@ -201,6 +203,52 @@ module.exports = function(simulatorMaker, api){
         var uploads = removeOverlaps(payload);
         progress(20);
 
+        function getUnits() {
+          var units = null;
```

Comment:
Yeah, good point. I did the getUnits function before I discovered exactly how many columns were going to have alternating names. I'll change this.